tests/Colore.Tests/Data/ColorTests.cs

```diff
@@ -229,8 +229,8 @@ public void ShouldNotEqualNull()
         {
             var c = new Color(255, 255, 255);
             Assert.AreNotEqual(c, null);
-            Assert.False(c == null);
-            Assert.True(c != null);
```

Comment:
For the `Assert` calls in test files, leave the `==` and `!=` as is, since these are testing operators defined on the types, and `is` / `is not` will bypass them.